python/ray/actor.py

```diff
@@ -140,6 +140,58 @@ def __setstate__(self, state):
             hardref=True)
 
 
+class ActorClassMethodMetadata(object):
+    """Metadata for all methods in an actor class. This data can be cached.
+
+    Attributes:
+        methods: The actor methods.
+        decorators: Optional decorators that should be applied to the
+            method invocation function before invoking the actor methods. These
+            can be set by attaching the attribute
+            "__ray_invocation_decorator__" to the actor method.
+        signatures: The signatures of the methods.
+        num_return_vals: The default number of return values for
+            each actor method.
+    """
+
+    def __init__(self, modified_class):
```

Comment:
Add a cache for this class? I think we can use a static method in this class.

Reply:
Implemented.